src/index.html

```diff
@@ -12,7 +12,49 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Use semantic tags where possible. For example, a `section` or `article` tag could be more appropriate for grouping the stars, depending on the context.